bindings/sampling-heap-profiler.cc

```diff
@@ -76,18 +73,48 @@ NAN_METHOD(StopSamplingHeapProfiler) {
 
 NAN_METHOD(GetAllocationProfile) {
   std::unique_ptr<v8::AllocationProfile> profile(
-    info.GetIsolate()->GetHeapProfiler()->GetAllocationProfile());
+      info.GetIsolate()->GetHeapProfiler()->GetAllocationProfile());
   AllocationProfile::Node* root = profile->GetRootNode();
   info.GetReturnValue().Set(TranslateAllocationProfile(root));
 }
 
+void free_buffer_callback(char* data, void* buf) {
+  delete reinterpret_cast<std::vector<char>*>(buf);
+}
+
+NAN_METHOD(GetAllocationProfileProto) {
```

Comment:
I don't see an actual call to this anywhere. Is that coming in a follow-on or did I miss it?

Reply:
Sorry -- I've fixed this, and this method is now used.